src/main/java/Player.java

```diff
@@ -1,23 +1,28 @@
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Player {
 
     private String name;
     private Coordinate coord;
-    private Item[] backpack;
-    private int score;
+    private List<Item> backpack;
     private int moves;
     private int health;
+    private static int MAX_HEALTH = 100;
```

Comment:
Consider adding final? You could also put the global constants in a separate class with public static final attributes so they are nicely combined in one place